### PR TITLE
Fix to compile and work with MSVC

### PR DESCRIPTION
**Pull request by smurfy**
Small changes to make it compile on MSVC.
Also tested if it actually work too with Win10 and Visual Studio 14 2015

### CODE REVIEW

File: src/jsonrpccpp/server/connectors/tcpsocketserver.cpp [hunk 1]
```diff
@@ -8,7 +8,7 @@
  ************************************************************************/
 
 #include "tcpsocketserver.h"
-#ifdef __WIN32__
```
Comment (ruslo) [inline src/jsonrpccpp/server/connectors/tcpsocketserver.cpp:11]: It's `_WIN32` actually:
* https://msdn.microsoft.com/en-us/library/b0084kay.aspx

**Review by ruslo — CHANGES_REQUESTED** [review]
Since this change is not about hunterization please send this patches upstream too.
* https://docs.hunter.sh/en/latest/creating-new/cmake-dependencies.html#maintenance

File: src/jsonrpccpp/server/connectors/tcpsocketserver.cpp (hunk 1 above)
Comment (smurfy) [reply src/jsonrpccpp/server/connectors/tcpsocketserver.cpp:11]: ok, force pushed new variant

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/libjson-rpc-cpp/releases/tag/v0.7.0-p1

**Pull request merged**